applications/dashboard/scss/legacy/_style.scss

```diff
@@ -2488,6 +2488,17 @@ dl.About dd {
   display: block
 }
 
+/* ======================================================== Image Cropper */
+body.isMobile {
```

Comment:
Don't forget you can also change your profile picture on the `/profile` page. You should also be hiding: `. ChangePicture` (link on the thumbnail) and `. PictureLink` in the panel.

Reply:
I don't see why we need to hide the page or the popup itself. Changing pictures works particularly well without the cropper and with things like the avatar pool.

Reply:
You're right, Keep the ability to change the picture, just hide the cropper.